features/mediaPlaybackRequiresUserGesture.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Files are named using `-` as separators so this file should be `media-playback-requires-user-gesture.json`